### ISSUE TITLE
Shot Generator has no features

### ISSUE THREAD

**Issue by kae55** [wonderunit/storyboarder#2503]
Hello,

I just downloaded Storyboarder today onto my Windows computer, and the shot generator feature is missing pretty much all of its features. The only character option is a generic male, and there are no pose presets. For objects there is only a box, and that has very limited adjustment options. Do I need to download separately to get all of the features that are supposed to be there?
Thanks

**Comment by cleargulf** [comment]
Yes.  I tried setting windows compatibility mode to Windows 8 and ran as administrator.  A lot of the missing features of shot generator appeared.   Posing, female figures, etc.  Magic.:)

**Comment by kae55** [comment]
Gave it a try, and it didn't change anything for me. All the features are still missing :(

**Comment by Rp2patilrahul** [comment]
i am on win 11 Pro & download Storyboarder-Setup-2.1.0 from website, all feature working for me 
see this -  https://snipboard.io/Uje0o3.jpg
